config.toml

```diff
@@ -11,3 +11,4 @@ enable_cors = false
 [services]
 aggregator = "http://localhost:8080/api/v1/"
 content = "http://localhost:8082/api/v1/"
+groups_poll_time = 60
```

Comment:
Why do we need to poll?

Reply:
We want to update the group configuration periodically to cache it. If we don't poll, we can miss groups configuration changes on the content service